### PR TITLE
remove v1beta3 examples from getting started from scratch guide

### PR DESCRIPTION
**Pull request by mikedanese**
(and formated) cc @nikhiljindal #10501

### CODE REVIEW

File: docs/getting-started-guides/scratch.md [hunk 1]
```diff
@@ -509,61 +509,73 @@ If you are using the HTTPS approach, then set:
...
-        "containerPort": 8080,
-        "hostPort": 8080}
-        ],
-    "volumeMounts": [
-        { "name": "srvkube",
-        "mountPath": "/srv/kubernetes",
-        "readOnly": true},
-        { "name": "etcssl",
-        "mountPath": "/etc/ssl",
-        "readOnly": true},
```
Comment (nikhiljindal) [inline docs/getting-started-guides/scratch.md]: You seem to have removed readOnly.
Is that intended?

File: docs/getting-started-guides/scratch.md [hunk 2]
```diff
@@ -658,49 +671,61 @@ Flags to consider using with controller manager.
...
+    "containers": [
+      {
+        "name": "kube-controller-manager",
+        "image": "$CNTRLMNGR_IMAGE",
+        "command": [
+          "/bin/sh",
+          "-c",
+          "/usr/local/bin/kube-controller-manager $ARGS"
+        ],
+        "resources": {},
```
Comment (nikhiljindal) [inline docs/getting-started-guides/scratch.md]: not required?

**Comment by nikhiljindal** [comment]
LGTM, modulo comments

**Comment by nikhiljindal** [comment]
@mikedanese Let me know when its ready for another look

**Comment by mikedanese** [comment]
@nikhiljindal yes PTAL.

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Comment by wojtek-t** [comment]
@davidopp for lead-LGTM

**Comment by vmarmol** [comment]
This is a doc-only change, so in theory we don't need a lead-LGTM. We already invoked the @davidopp so maybe we'll just wait :)

Will merge with the ok-to-merge

**Comment by davidopp** [comment]
LGTM

**Pull request merged**